src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectSecurityAwarenessScore.java

```diff
@@ -87,6 +90,10 @@
  *     {@link com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures
  *     #USES_UNDEFINED_BEHAVIOR_SANITIZER}
  *   </li>
+ *   <li>
```

Comment:
What is the difference between the list line and the line here https://github.com/SAP/fosstars-rating-core/pull/329/files#diff-34455b8acb52a5e62a987dc931534d8cR24?

Is it not possible to update this list as similar to the one in https://github.com/SAP/fosstars-rating-core/pull/329/files#diff-34455b8acb52a5e62a987dc931534d8cR24

Reply:
Someone may not like adding imports just for javadoc.

Reply:
https://github.com/SAP/fosstars-rating-core/pull/252#discussion_r450840948